include/contraction/deadEndContraction.hpp

```diff
@@ -94,15 +94,15 @@ class Pgr_deadend {
                  * u{v1 + v + v2 + v3}     v{}
                  */
                 auto v2(graph.get_min_cost_edge(u, v));
-                graph[u].contracted_vertices() += std::get<1>(v2);
-                graph[u].contracted_vertices() += graph[v].id;
-                graph[u].contracted_vertices() += graph[v].contracted_vertices();
+                graph[u].get_contracted_vertices() += std::get<1>(v2);
+                graph[u].get_contracted_vertices() += graph[v].id;
+                graph[u].get_contracted_vertices() += graph[v].get_contracted_vertices();
```

Comment:
I'm wondering: could this be managed through your `add_contracted_vertex_id` method?